### ISSUE TITLE
Add generic BT command learning and sending ability

### ISSUE THREAD

**Issue by iz3man** [esphome/feature-requests#579]
**Describe the problem you have/What new integration you would like**
I'd like to be able to READ a BT command sent from my iphone and then let esphome SEND it again. 

**Please describe your use case for this integration and alternatives you've tried:**
I got a terrace ir heater which is controlled by BT only. An app for iphone/android exists which can control the heater. It would be great to have another way of controlling the heater to integrate it into home automation.
A generic learn function of a BT signal and sending it out could solve that.
I have NO idea if BT works that way and if this can be done though ;)

**Additional context**
This is the product I'm talking about:
https://www.amazon.de/gp/product/B00YE642U0/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

**Comment by torfbolt** [comment]
No, unfortunately BT does not work this way. Reverse engineering a BT interface is a lot harder than e.g. an infrared remote control signal. A simple replay does not work here.
